### PR TITLE
-- grammar

### PR DESCRIPTION
**Pull request by bluepeter**

### CODE REVIEW

File: content/recipes/migration.mdx [hunk 1]
```diff
@@ -17,11 +17,11 @@ The library has been internally rewritten to use new Context. You will need Reac
 
 ### What about mobx-react 5.x?
 
-If you are forced to stick to the older version (most likely because older React), there isn't really anything you can do right now and just keep using inject as you are used to.
+If you are forced to stick to the older version (most likely because oof lder React), there isn't really anything you can do right now and just keep using inject as you are used to.
```
Comment (FredyC) [inline content/recipes/migration.mdx]: Second honest typo? :) Not sure how you be fixing something and making it actually worse :)

File: content/recipes/migration.mdx [hunk 2]
```diff
@@ -48,19 +48,19 @@ function useStores() {
...
 
-You can build on top of that and make a solution that fits your needs. For example have a hook for each separate store, eg. `useUserStore` or `useOrderStore`. It's a similar to passing string to inject, but much safer.
+You can build on top of that and make a solution that fits your needs. For example, have a hook for each separate store: e.g., `useUserStore` or `useOrderStore`. It's similar to passing string to `inject`, but much safer.
 
-### Why the hook is not exported from package?
+### Why the hook is not exported from the package?
 
-It's becoming a good practice for packages to always hide the Context object and only export a relevant hook. Usually the reason are to abstract some internals tucked away in context. However, in case of MobX, there is nothing to hide, it's stores you have added there.
+It's becoming a good practice for packages to always hide the Context object and only export a relevant hook. Usually the reasons are to abstract some internals tucked away in context. However, in case of MobX, there is nothing to hide, it's stores you have added there.
```
Comment (FredyC) [inline content/recipes/migration.mdx]: I think I actually want to drop the whole paragraph, it's not that relevant and more confusing.
```suggestion
```

File: content/recipes/migration.mdx [hunk 3]
```diff
@@ -48,19 +48,19 @@ function useStores() {
...
 
-Think of it more like tool for migration. The concept of Context is so simple that there is no added value of having it inside the library. If you ever manage to migrate a whole app to hooks, you can drop the Provider from `mobx-react` and use your own to gain even more control about it. Besides, it's inherently easier to correct types for own context (eg. with TypeScript).
+Think of it more like a tool for migration. The concept of Context is so simple that there is no added value of having it inside the library. If you ever manage to migrate a whole app to hooks, you can drop the Provider from `mobx-react` and use your own to gain even more control. Besides, it's inherently easier to correct types for own context (e.g., with TypeScript).
```
Comment (FredyC) [inline content/recipes/migration.mdx]:
```suggestion
Think of it more like a tool for migration. The concept of Context is so simple that there is no added value of having it inside the library. If you ever manage to migrate a whole app to hooks, you can drop the Provider from `mobx-react` and use your own to gain even more control. Besides, it's inherently easier to correctly declare types for own context (e.g., with TypeScript).
```

File: content/recipes/migration.mdx (hunk 1 above)
Comment (bluepeter) [reply content/recipes/migration.mdx]: Oh man, this is not good! Fixed now.

File: content/recipes/migration.mdx (hunk 2 above)
Comment (FredyC) [reply content/recipes/migration.mdx]: You need to click on Commit suggestion since you haven't checked "Allow edit for maintainers".

**Pull request merged**